lib/driver.js

```diff
@@ -122,7 +122,7 @@ class XCUITestDriver extends BaseDriver {
       this.wda = new WebDriverAgent({
         sim: this.sim, //TODO can't be a sim if real device. find a different way to supply logs to webdriveragent
         platformVersion: this.opts.platformVersion,
-        host: this.opts.host,
+        host: this.opts.address,
```

Comment:
when used in conjunction with appium top level project we use 'address' in our parsed args [here](https://github.com/appium/appium/blob/master/lib/parser.js#L23). Do you think we should support both so this also works when the driver is used standalone? for example:

```
host: this.opts.host || this.opts.address,
```